do.sh

```diff
@@ -200,6 +211,12 @@ map <down> <nop>
 map <left> <nop>
 map <right> <nop>
 
+" Change move between windows on cmder "
```

Comment:
What does this do?

Reply:
When you use ctrl-w short cut, was dispatch close event on cmdr, I set new short cuts for a move between NERDTRee window and editor window.